src/main/java/teammates/common/util/Const.java

```diff
@@ -931,15 +887,13 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String AUTOMATED_FEEDBACK_OPENING_REMINDERS = "/feedbackSessionOpeningReminders";
         public static final String AUTOMATED_FEEDBACK_CLOSING_REMINDERS = "/feedbackSessionClosingReminders";
         public static final String AUTOMATED_FEEDBACK_PUBLISHED_REMINDERS = "/feedbackSessionPublishedReminders";
-        public static final String AUTOMATED_COMPILE_LOGS = "/compileLogs";
```

Comment:
Added in cbd739f. The URL is used in `web.xml`, but not the constant, so likely it's a case where it's there just for the sake when it's going to be needed in the future.